schemas/com.snowplowanalytics.accelerators.travel/add_to_itinerary/jsonschema/1-0-0

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "type": "object",
+  "self": {
+    "vendor": "com.snowplowanalytics.accelerators.travel",
+    "name": "add_to_itinerary",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "description": "Event for when an item has been added to the trip itinerary",
+  "properties": {
+    "type": {
+      "type": "string",
+      "description": "Type of item added (e.g. hotel, experience, flight)",
+      "maxLength": 128
+    },
+    "price": {
+      "type": [
+        "number",
```

Comment:
For our structured warehouses (everything apart from snowflake) this will get stored in the warehouse as a double.  For monetary fields in iglu central we often add `multipleOf: 0.01` so that it gets stored in the warehouse with decimal type.